### ISSUE TITLE
[Epic] Debts (debt portal, FSR, financial hardship, and related pages)

### ISSUE THREAD

**Issue by strelichl** [department-of-veterans-affairs/va.gov-team#81479]
This epic will house work related to FSR wizard deprecation and transition to complete a subtask. 

The tickets are organized by intake:
**Intake title as task list title**
- intake ticket
- associated child tickets (linked to epic, not nested under intake ticket)

```[tasklist]
### Support for FSR Complete a Subtask
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/71331
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/72088
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/74776
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/73914
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/73909
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/73145
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/87307
```

```[tasklist]
### FSR form and Health Care page updates
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/69414
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/79247
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/78039
```

```[tasklist]
### Other debt-related work
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/83243
```

**Comment by laurwill** [comment]
@strelichl @NaomiPMC I realized we already had this epic going for debt portal related work, so I closed the new "debt portal" epic and am moving all payments & debts tickets here. Lmk if this causes any issues!

**Comment by coforma-jamie** [comment]
> [!NOTE]
> ## Accessibility Digital Experience (DE) Ticket Status Update
> Accessibility DE is OCTO’s new accessibility team. We’re working on a new intake process. In the meanwhile:
> - This ticket has been moved to validate and will be closed on the Accessibility DE pipeline (only).
> - This ticket will remain “open” until the work from content and IA is completed.
> 
> If you need additional accessibility support, contact the Accessibility DE team on [#accessibility-help](https://dsva.slack.com/archives/C8E985R32).
> **Thank you for working with the accessibility team!**

**Comment by mnorthuis** [comment]
closing product-specific epics
